http/http-minimum-reactive/src/main/java/io/quarkus/ts/http/minimum/reactive/OperatorResource.java

```diff
@@ -0,0 +1,27 @@
+package io.quarkus.ts.http.minimum.reactive;
+
+import jakarta.interceptor.Interceptors;
+import jakarta.ws.rs.Consumes;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+
+import io.smallrye.mutiny.Uni;
+
+/**
+ * The OperatorResource.
```

Comment:
This is absolutely optional, but I could do without this javadoc. Can't say it gives me any information (same goes for JakartaRestInterceptor)